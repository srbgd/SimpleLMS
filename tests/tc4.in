

drop
register librarian --login l --password pass --name test --address innopolis --phone-number 88005553535 --card-number 1234
register faculty --login f --password pass --name test --address innopolis --phone-number 88005553535 --card-number 1234
register student --login s --password pass --name test --address innopolis --phone-number 88005553535 --card-number 9876
insert best-seller --title b title --authors unknown --publisher empty --edition none --date unknown --price 1000
copy 3
login
f
pass
checkout 3
login
l
pass
check
exit
