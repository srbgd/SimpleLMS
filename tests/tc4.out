Success
True
True
True
True
True
True
Success
True
Success
[{'attributes': {'deadline': '06/03/2018', 'origin_id': '3', 'user_id': 1}, 'type': 'copy', 'id': 4}]
